versioned_docs/version-1.2/index.md

```diff
@@ -29,7 +29,7 @@ SLE Micro for Rancher is a containerized and "stripped to the bones" OS layer. A
 It's sole purpose is to run Kubernetes (k3s or RKE2), with everything controlled through Rancher Manager.
 
 Elemental Teal is built in the [openSUSE Build Service](https://build.opensuse.org/package/show/isv:Rancher:Elemental:Stable:Teal53/node-image)
-and available through the [openSUSE Registry](https://registry.opensuse.org/cgi-bin/cooverview?srch_term=project%3D%5Eisv%3ARancher%3AElemental%3AStable+container%3D.*).
+and available through the [SUSE Registry](https://registry.suse.com).
```

Comment:
Same as above, I doubt this is a good way to link it.

Reply:
I would leave also the OBS reference as stated above...
apart from this, what about giving the link to the teal image repo in the registry?
oci://registry.suse.com/rancher/elemental-teal/5.3

One could inspect / list the tags at least, e.g., with skopeo:
`skopeo list-tags docker://registry.suse.com/rancher/elemental-teal/5.3`
the oci link may prevent misunderstandings.

Reply:
OBS reference is good, as it points to the source code. Listing available tags from an skopeo/reg call sounds good to me too. Probably `reg` as it is also available for Mac, Windows and Linux.